### ISSUE TITLE
Importer: Error units update problem

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.Core#1428]
[TestData.xlsx](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/files/7828854/TestData.xlsx)

In MoBi:

- Import the sheet "Mitrakou_1990". Change the mapping of "Measurement" to "Concentration (molar) [µmol/l]"

![image](https://user-images.githubusercontent.com/25903040/148546385-267369d7-3579-4a69-9480-bd8a040149ec.png)

The "Units" field shows an error. Clicking on the field "Edit extra fields" shows a view with "µmol/l" as unit, but the error is still present

![image](https://user-images.githubusercontent.com/25903040/148546487-25a631c6-2ab9-4a11-8715-71fbd31bcb10.png)

Only after selecting another unit (e.g. "nmol/l") and then back to "µmol/l", the error disappears.

**Comment by abdelr** [comment]
This was actually an error introduced with the addition of the dimension to the settings. It has already been solved in #1434

**Comment by PavelBal** [comment]
Re-opening as it is not really fixed - see https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1427 for description.

**Comment by abdelr** [comment]
Closing in favor of #1427
